app/Modules/Bookings/CreateBookingHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Modules\Bookings;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Nuwave\Lighthouse\Support\Contracts\GraphQLContext;
+
+class CreateBookingHandler
+{
+    /**
+     * Return a value for the field.
+     *
+     * @param  null  $rootValue  Usually contains the result returned from the parent field. In this case, it is always `null`.
+     * @param  mixed[]  $args  The arguments that were passed into the field.
+     * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext  $context  Arbitrary data that is shared between all fields of a single query.
```

Comment:
Could `use` these so we dont have to have them full length

Reply:
done!